docs/querying/sql-json-functions.md

```diff
@@ -0,0 +1,71 @@
+---
+id: sql-json-functions
+title: "SQL JSON functions"
+sidebar_label: "JSON functions"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<!--
+  The format of the tables that describe the functions and operators
+  should not be changed without updating the script create-sql-docs
+  in web-console/script/create-sql-docs, because the script detects
+  patterns in this markdown file and parse it to TypeScript file for web console
+-->
+
+Druid supports nested columns, which provide optimized storage and indexes for nested data structures. These JSON
+functions provide facilities to extract, transform, and create `COMPLEX<json>` values.
+
+| function | notes |
```

Comment:
```suggestion
| Function | Notes |
```
Capitalization throughout the docs is not standardized but default towards sentence case in table headings